odoo-shippable/scripts/build-image.sh

```diff
@@ -187,7 +187,8 @@ EOF
 # Install and configure YouCompleteMe
 VIM_YOUCOMPLETEME_PATH="${HOME}/.vim/bundle/YouCompleteMe"
 git_clone_copy "${VIM_YOUCOMPLETEME_REPO}" "master" "." "${VIM_YOUCOMPLETEME_PATH}"
-(cd "${VIM_YOUCOMPLETEME_PATH}" && ./install.py)
+# The version required by youcompleteme is g++4.9
+#(cd "${VIM_YOUCOMPLETEME_PATH}" && ./install.py)
```

Comment:
What about use a previous version of youcompleteme where don't requires a update of g++?